packages/garbo/src/config.ts

```diff
@@ -83,12 +83,16 @@ function stringToWorkshedItem(s: string): Item | null {
   return validWorksheds[0].item;
 }
 
+// We want this to be a const because we don't want to determine we can no longer target Cheerless Mime Executive simply because we used all reminisces to start a chain
+const locketAvailable = CombatLoversLocket.reminiscesLeft() > 0;
```

Comment:
so this check ensures that you have a reminisce left that can be used before declaring it to be your way to start the chain. makes sense, reasonable addition.

Reply:
dont' define this as a const outside of the thing, just toss it into the big pile of ternary unless you're going to do that for everything

Reply:
We want it to be a const because the value of it could change over the course of the garbo run, and we don't want to accidentally change the target during the course of the garbo run. Otherwise, every time we call the function (which should be, but may not be, exactly once) we run the risk of changing targets.